lib/model/query/analytics.js

```diff
@@ -420,103 +420,116 @@ group by f."projectId"`);
+      GROUP BY e."datasetId"
+    ) as creates ON ds.id = creates."datasetId"
+    LEFT JOIN (
+      SELECT COUNT(*) total,
+      SUM (CASE WHEN a."loggedAt" >= current_date - 45 THEN 1 ELSE 0 END) recent,
       e."datasetId"
-    FROM audits a
-      JOIN entities e on CAST((a.details ->> 'entityId'::TEXT) AS integer) = e.id
-    WHERE a."action" = 'entity.update.version'
-    GROUP BY e."datasetId"
-  ) as updates ON ds.id = updates."datasetId"
-  LEFT JOIN (
-    SELECT
-      COUNT (1) total,
-      SUM (CASE WHEN a."loggedAt" >= current_date - cast(${DAY_RANGE} as int) THEN 1 ELSE 0 END) recent,
-      SUM (CASE WHEN a."details"->'submissionDefId' IS NOT NULL THEN 1 ELSE 0 END) create_sub_total,
-      SUM (CASE WHEN a."details"->'submissionDefId' IS NOT NULL AND a."loggedAt" >= current_date - cast(${DAY_RANGE} as int)
-        THEN 1 ELSE 0 END) create_sub_recent,
-      SUM (CASE WHEN a."details"->'submissionDefId' IS NULL THEN 1 ELSE 0 END) create_api_total,
-      SUM (CASE WHEN a."details"->'submissionDefId' IS NULL AND a."loggedAt" >= current_date - cast(${DAY_RANGE} as int)
-        THEN 1 ELSE 0 END) create_api_recent,
+      FROM audits a
+        JOIN entity_def_sources eds on CAST((a.details ->> 'sourceId'::TEXT) AS integer) = eds."id"
+        JOIN entity_defs ed on ed."sourceId" = eds.id AND root=true
+        JOIN entities e on ed."entityId" = e.id
+      WHERE a."action" = 'entity.bulk.create'
+      GROUP BY e."datasetId"
+    ) as bulk_creates on ds.id = bulk_creates."datasetId"
+    LEFT JOIN (
+      SELECT COUNT(*) conflicts,
+      SUM (CASE WHEN e."conflict" IS NULL THEN 1 ELSE 0 END) resolved,
       e."datasetId"
-    FROM audits a
-      JOIN entities e on CAST((a.details ->> 'entityId'::TEXT) AS integer) = e.id
-    WHERE a."action" = 'entity.create'
-    GROUP BY e."datasetId"
-  ) as creates ON ds.id = creates."datasetId"
-  LEFT JOIN (
-    SELECT count(1) total,
-    SUM (CASE WHEN a."loggedAt" >= current_date - cast(${DAY_RANGE} as int) THEN 1 ELSE 0 END) recent,
-    e."datasetId"
-    FROM audits a
-      JOIN entity_def_sources eds on CAST((a.details ->> 'sourceId'::TEXT) AS integer) = eds."id"
-      JOIN entity_defs ed on ed."sourceId" = eds.id AND root=true
-      JOIN entities e on ed."entityId" = e.id
-    WHERE a."action" = 'entity.bulk.create'
-    GROUP BY e."datasetId"
-  ) as bulk_creates on ds.id = bulk_creates."datasetId"
-  LEFT JOIN (
-    SELECT COUNT (1) conflicts,
-    SUM (CASE WHEN e."conflict" IS NULL THEN 1 ELSE 0 END) resolved,
-    e."datasetId"
-    FROM entities e
-    WHERE e.id IN
-      (SELECT DISTINCT "entityId" FROM entity_defs WHERE "conflictingProperties" IS NOT NULL)
-    GROUP BY e."datasetId"
-  ) AS conflict_stats ON ds.id = conflict_stats."datasetId"
-WHERE ds."publishedAt" IS NOT NULL
-GROUP BY ds.id, ds."projectId"
+      FROM entities e
+      WHERE e.id IN
+        (SELECT DISTINCT "entityId" FROM entity_defs WHERE "conflictingProperties" IS NOT NULL)
```

Comment:
Since execution is <1s this is a bit of a golf, but in the plan file line 463 has Rows Removed by Filter: 123490 which I think could be avoided by replacing
```WHERE e.id IN (SELECT DISTINCT "entityId" FROM entity_defs WHERE "conflictingProperties" IS NOT NULL)```
with
```WHERE EXISTS (SELECT 1 FROM entity_defs AS ed WHERE ed."conflictingProperties" IS NOT NULL AND ed."entityId" = e.id)```
since the latter doesn't imply materialising the IDs for `IN` or the outer `Sort` and `Unique` plan nodes.